packages/core/test/unit/patchers/aws3_p.test.js

```diff
@@ -74,9 +74,34 @@ describe('AWS v3 patcher', function() {
         },
         middlewareStack: constructStack(),
       };
+
+      ddbClient = {
+        send: async (req) => {
+          const context = {
+            clientName: 'DynamoDBClient',
+            commandName: 'GetItemCommand',
```

Comment:
Based on the `awsClient` above, which has a `ListBucketsCommand` in the `context` variable, should this `commandName` be `PutItemCommand` to match the new `ddbGetRequest` added below?

Reply:
This is fine, what needed to be changed is the class name for the request further down